python/defs.bzl

```diff
@@ -47,6 +47,8 @@ def py_library(**attrs):
     Args:
       **attrs: Rule attributes
     """
+
+    # buildifier: disable=native-python
```

Comment:
Can we disable this just once for the whole file? (It'd be appropriate given the purpose of this file.)

Reply:
Looks like we can't.